subsys/bluetooth/audio/ascs.c

```diff
@@ -1158,12 +1158,20 @@ static void ase_stream_add(struct bt_ascs *ascs, struct bt_ascs_ase *ase,
 	stream->ep = &ase->ep;
 }
 
+static void ascs_init(struct bt_ascs *ascs, struct bt_conn *conn)
+{
+	memset(ascs, 0, sizeof(*ascs));
+
+	ascs->conn = bt_conn_ref(conn);
+	sys_slist_init(&ascs->ases);
```

Comment:
If we are now `memset`ing `ascs`, then `sys_slist_init(&ascs->ases);`, regardless of where the `ascs` is stored, won't do anything.

Reply:
Yeah, but what if `sys_slist_init` will be changed some day to do more job than it does now (I know it's unlikely)?

Reply:
> Yeah, but what if `sys_slist_init` will be changed some day to do more job than it does now (I know it's unlikely)?

True, it's fine to add this.